web_src/js/features/common-form.js

```diff
@@ -1,4 +1,5 @@
 import $ from 'jquery';
+import '../vendor/jquery.are-you-sure.js';
```

Comment:
Already imported in `web_src/js/features/common-global.js`, so would load it twice. Move import to index.js?

Reply:
There is no `common-global.js`

Reply:
Where is it currently loaded? Not at all?

Reply:
> Move import to index.js?

I would say no. Only use it where it is really used. Here is initGlobalFormDirtyLeaveConfirm

Reply:
https://github.com/go-gitea/gitea/commit/06782872c42af7450e72527e9e9cd83eacd467b9 regressed it. I recommend moving the import to top of index.js, it's needed on so many pages, we may as well load it everywhere.

Reply:
->  Fix are-you-sure #31446

Reply:
Maybe we do not need that many PRs, merged #31446 into this.